.github/ISSUE_TEMPLATE/BugReport.yml

```diff
@@ -0,0 +1,43 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+labels: ["bug, invalid"]
```

Comment:
One item per label – it should be `["bug", "invalid"]`. Besides, `"invalid"` means that this issue is spam or not discussable; therefore, I suggest removing it.